sse2neon.h

```diff
@@ -6025,7 +6023,13 @@ FORCE_INLINE __m128d _mm_addsub_pd(__m128d a, __m128d b)
 FORCE_INLINE __m128 _mm_addsub_ps(__m128 a, __m128 b)
 {
     __m128 mask = {-1.0f, 1.0f, -1.0f, 1.0f};
-    return _mm_fmadd_ps(b, mask, a);
+#if defined(__aarch64__)
+    return vreinterpretq_m128_f32(vfmaq_f32(vreinterpretq_f32_m128(a),
+                                            vreinterpretq_f32_m128(mask),
+                                            vreinterpretq_f32_m128(b)));
+#else
+    return _mm_add_ps(_mm_mul_ps(b, mask), a);
+#endif
```

Comment:
Remove the ARM 32-bit implementation since `vfmaq_f32` is also supported in A32.

Reply:
`vfmaq_f32` is only available for VFPv4+. Thus, for Armv7-A targets, we have to take the following cases into consideration:
- VFPv3, which is implemented on Cortex-R4 and R5 processors and the Tegra 2 (Cortex-A9).
- VFPv4, which is implemented on the A15 and Cortex-A7, or later

Reference: https://embeddedartistry.com/blog/2017/10/11/demystifying-arm-floating-point-compiler-options/